march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -230,6 +231,12 @@ void MarchHardwareInterface::write(const ros::Time& /* time */, const ros::Durat
   {
     // Enlarge joint_effort_command because ROS control limits the pid values to a certain maximum
     joint_effort_command_[i] = joint_effort_command_[i] * 1000.0;
+    if (abs(joint_effort_command_copy_[i] - joint_effort_command_[i]) > MAX_EFFORT_CHANGE)
+    {
+      joint_effort_command_copy_[i] =
+          joint_effort_command_copy_[i] +
+          copysign(MAX_EFFORT_CHANGE, joint_effort_command_[i] - joint_effort_command_copy_[i]);
```

Comment:
Instead of using this if and copysign structure, you could also use 
`joint_effort_command_[i] = std::min(joint_effort_command_copy_[i] + MAX_EFFORT_CHANGE, std::max(joint_effort_command_copy_[i] - MAX_EFFORT_CHANGE), joint_effort_command_[i])`

Reply:
Why would this be better?

Reply:
Was intended as a suggestion. Use whatever has your preference :+1: